src/components/CreateResetPasswordLoader/CreateResetPasswordLoaderCtrl.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import {
+  withRouter,
+  Redirect,
+} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { isArray } from 'lodash';
+
+import CreateResetPasswordLoader from './CreateResetPasswordLoader';
+import {
+  defaultErrorCodes,
+} from '../../constants';
+import { stripesShape } from '../../Stripes';
+
+class CreateResetPasswordLoaderCtrl extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        token: PropTypes.string.isRequired,
+      }).isRequired,
+    }).isRequired,
+    stripes: stripesShape.isRequired,
+  };
+
+  state = {
+    isLoaded: false,
+    isValidToken: false,
+    errorCodes: [],
+    actionId: 0,
+  };
+
+  async componentDidMount() {
+    const {
+      match: {
+        params: {
+          token,
+        },
+      },
+      stripes: {
+        okapi: {
+          url,
+          tenant,
+        },
+      },
+    } = this.props;
+
+    try {
+      const response = await fetch(`${url}/bl-users/password-reset/validate`, {
```

Comment:
Do we have a reason why we use fetch instead of stripes mutator?

Reply:
We have to pass the token in headers and Zak approved such an approach